projects/lkft-android/master.jinja2

```diff
@@ -0,0 +1,68 @@
+{% extends "testcases/android/master.jinja2" %}
```

Comment:
Can I generate an android job file without any project set?

Reply:
yes, the default selftests.sh check will have no project specified.